google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketPanel.java

```diff
@@ -69,6 +79,21 @@ protected void textChanged(DocumentEvent event) {
                 refresh();
               }
             });
+
+    bucketList.addMouseListener(
+        new MouseAdapter() {
+          @Override
+          public void mouseClicked(MouseEvent event) {
+            if (event.getClickCount() == 2) {
```

Comment:
There's really no double-click listener? Poor Swing :(

Reply:
yeah, I thought the same thing ..